### ISSUE TITLE
Make scroll container height fit to content height if smaller

### ISSUE THREAD

**Issue by Aphax** [rintoj/angular2-virtual-scroll#254]
I'm trying to make the scroll container's height fit to its content's height if the latter is smaller than him.
I can not see any option for this behavior to be handled actually, is there any planned ?

Only solution I see for now is to set container height manually after scroll is updated by calculating its content height and do the comparison manually.

Thanks for the good work already

**Comment by speige** [comment]
You're correct, there is no built-in functionality for this in the component.
If you'd like to contribute this feature, feel free to submit a pull request.

**Comment by orhemi** [comment]
Try to add #container, it worked fo me

**Comment by DinoSourcesRex** [comment]
@orhemi Any chance of a stack-blitz example of this? I've got a similar issue I need to resolve.
